Nos/Service/FileStorage/FileStorageAPIClient.swift

```diff
@@ -59,9 +60,26 @@ class NostrBuildAPIClient: FileStorageAPIClient {
         let (responseData, _) = try await URLSession.shared.upload(for: request, from: data)
 
         let response = try decoder.decode(FileStorageUploadResponseJSON.self, from: responseData)
+        
         guard let urlString = response.nip94Event?.urlString else {
-            throw FileStorageAPIClientError.uploadFailed(response.message)
+            // Ensure there's a message in the response, use an empty string if not
+            let message = response.message ?? ""
+
+            // Check to see if the error message mentions the file size
```

Comment:
Is there another way to tell, like the HTTP status code? If their messaging were to change, this could break, but hopefully they send a HTTP 413 Payload Too Large and we could use that to display our message.

Reply:
Fxied